setup.py

```diff
@@ -518,17 +518,25 @@ def finalize_options(self):
         # To support the package consisting of both openvino and training modules part of it
         package_name = "onnxruntime-training"
 
-    # we want put default training packages to pypi. pypi does not accept package with a local version.
-    if not default_training_package_device or nightly_build:
-        if cuda_version:
-            # removing '.' to make Cuda version number in the same form as Pytorch.
-            local_version = "+cu" + cuda_version.replace(".", "")
-        elif rocm_version:
-            # removing '.' to make Rocm version number in the same form as Pytorch.
-            local_version = "+rocm" + rocm_version.replace(".", "")
-        else:
-            # cpu version for documentation
-            local_version = "+cpu"
+    disable_local_version = environ.get("DISABLE_LOCAL_VERSION", "0")
```

Comment:
if we are using an environment variable, can we give it a very specific name with a prefix? e.g., "ORT_DISABLE_PYTHON_PACKAGE_LOCAL_VERSION"